pkg/kapp/version/version.go

```diff
@@ -3,6 +3,4 @@
 
 package version
 
-const (
-	Version = "0.39.0"
-)
+var Version = "develop"
```

Comment:
I guess we do not wanna update this value manually, but is this change required for the automation of the release process? This variable seems to be used in a number of places and is parsed by `semver` in `pkg/kapp/semver.go` as well. This will cause kapp to error out if that code path is traversed. We should ensure this change does not affect other usages?

Reply:
> but is this change required for the automation of the release process

It is currently a manual step to update the version number before releasing a new version.

> We should ensure this change does not affect other usages

Sure, if you can identify any other usages that would break from this, definitely please list them out here. Also I think adding test coverage for those cases will help with future discoverability.